prosemirror-jdita/src/plugin.ts

```diff
@@ -137,18 +136,18 @@ export function menu(schema: Schema, { start, before, after, end}: Additions = {
     }, { label: 'Show debug info', class: 'ic-bug', css: 'color: #c81200' }),
   ];
   const toolbar:MenuElement[][] = [[
-    commandItem(undo, { icon: {}, title: 'Undo', class: 'ic-undo' }),
-    commandItem(redo, { icon: {}, title: 'Redo', class: 'ic-redo' }),
+    commandItem(undo, { title: 'Undo', class: 'ic-undo' }),
+    commandItem(redo, { title: 'Redo', class: 'ic-redo' }),
   ], [
-    markItem(schema.marks.b, { icon: {}, title: 'Bold', class: 'ic-bold' }),
```

Comment:
the default  value for icon is no longer an empty object.
and it's optional now, so i decided on removing it